src/main/java/com/rapid7/appspider/ScanManagement.java

```diff
@@ -3,36 +3,38 @@
 import org.json.JSONObject;
 import org.json.JSONArray;
 
+import java.util.Arrays;
 import java.util.HashMap;
 import java.util.Map;
+import java.util.Optional;
 
 /**
  * Created by nbugash on 09/07/15.
  */
 public class ScanManagement extends Base {
 
-    private final static String GETSCANS = "/Scan/GetScans";
-    private final static String RUNSCAN = "/Scan/RunScan";
-    private final static String CANCELSCAN = "/Scan/CancelScan";
-    private final static String PAUSESCAN = "/Scan/PauseScan";
-    private final static String RESUMESCAN = "/Scan/ResumeScan";
-    private final static String PAUSEALLSCANS = "/Scan/PauseAllScans";
-    private final static String STOPALLSCANS = "/Scan/StopAllScans";
-    private final static String RESUMEALLSCANS = "/Scan/ResumeAllScans";
-    private final static String CANCELALLSCANS = "/Scan/CancelAllScans";
-    private final static String GETSCANSTATUS = "/Scan/GetScanStatus";
-    private final static String ISSCANACTIVE = "/Scan/IsScanActive";
-    private final static String ISSCANFINISHED = "/Scan/IsScanFinished";
-    private final static String HASREPORT = "/Scan/HasReport";
-    private final static String GETSCANERRORS = "/Scan/GetScanErrors";
+    private final static String GET_SCANS = "/Scan/GetScans";
```

Comment:
This is much better, easy to read

Reply:
yeah, I can see what they were going but it made it tricky to read, gotta pick a standard and stick with it